lib/SILOptimizer/Mandatory/DiagnoseUnreachable.cpp

```diff
@@ -79,20 +80,20 @@ class UnreachableUserCodeReportingState {
   ///
   /// This is a SetVector since several blocks may lead to the same error
   /// report and we iterate through these when producing the diagnostic.
-  llvm::SetVector<const SILBasicBlock*> PossiblyUnreachableBlocks;
+  llvm::SetVector<SILBasicBlock*> PossiblyUnreachableBlocks;
 
   /// The set of blocks in which we reported unreachable code errors.
   /// These are used to ensure that we don't issue duplicate reports.
   ///
   /// Note, this set is different from the PossiblyUnreachableBlocks as these
   /// are the blocks that do contain user code and they might not be immediate
   /// successors of a folded branch.
-  llvm::SmallPtrSet<const SILBasicBlock*, 2> BlocksWithErrors;
+  llvm::SmallPtrSet<SILBasicBlock*, 2> BlocksWithErrors;
```

Comment:
Seems like maybe you forgot to finish fixing this part? Shouldn't this be a BasicBlockSet?

Reply:
Indeed. I changed it